### ISSUE TITLE
Pictures and measures of the device disassembled

### ISSUE THREAD

**Issue by joaomorenorf** [Xinyuan-LilyGO/LilyGO-Mini-Epaper#2]
I'm thinking of disassembling the case and epoxying the PCB.
Pictures of the device totally opened would really help me to understand if it's feasible. Measures without the case would help a lot too.
Thank you!

**Comment by LilyGO** [comment]
Hello . Are you trying to figure out how to remove the housing?

**Comment by joaomorenorf** [comment]
No, I want to see only the PCB, without the case. I plan to clear epoxy the PCB to do like a pendant, but I want to know if I can fit a micro battery inside.
